chi/server.py

```diff
@@ -785,7 +787,7 @@ def show_server_by_name(name) -> NovaServer:
     return show_server(server_id)
 
 
-def associate_floating_ip(server_id, floating_ip_address=None):
+def associate_floating_ip(server_id, floating_ip_address=None, port_id=None):
     """
     .. deprecated:: 1.0
```

Comment:
Is this still deprecated?

Reply:
Yes, it's now just a helper for Server.associate_floating_ip.